public/doc/openapi-3-v1.1.json

```diff
@@ -3181,9 +3181,10 @@
               "undecided",
               "approved",
               "denied",
-              "canceled"
+              "canceled",
+              "error"
```

Comment:
Just noting we need this change for Order and OrderItem as well.  Ok if it is done in the followup PR for model enums.

Reply:
@gmcculloug for order and order items its called "Failed"